gulpfile.js

```diff
@@ -323,7 +323,7 @@ function buildJsPipeline(assetGroup, doConcat, doRebuild) {
             "** NOTE: This file is generated by Gulp and should not be edited directly!\n" +
             "** Any changes made directly to this file will be overwritten next time its asset group is processed by Gulp.\n" +
             "*/\n\n"))
-        .pipe(gulpif(generateSourceMaps, sourcemaps.write()))
+        .pipe(gulpif(generateSourceMaps, sourcemaps.write(assetGroup.outputDir)))
```

Comment:
@Skrypt this change has broken building the media app 

or I think any assets that have a comma delimited listing

i.e.
```
    "inputs": [
      "./node_modules/blueimp-file-upload/js/jquery.fileupload.js",
      "./node_modules/blueimp-file-upload/js/jquery.iframe-transport.js",
      "Assets/js/**/*.js"
    ],
```

what's the change for?

Reply:
To be able to generate a sourceMap file. This is not supposed to be related.

Reply:
I think someone added a line in the gulpfile.cs so that the Media app be taken into consideration recently.